WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift

```diff
@@ -0,0 +1,273 @@
+            .padding(.horizontal, Layout.contentPadding)
+
+            AttributedText(agreementText)
+                .padding(.horizontal, Layout.contentPadding)
+        }
+        .padding(.vertical, Layout.contentPadding)
+        .background(Color(.systemBackground))
+    }
+}
+
+private extension BlazeConfirmPaymentView {
+
+    enum Layout {
+        static let contentPadding: CGFloat = 16
+    }
+
+    enum Constants {
+        static let termsLink = "https://wordpress.com/tos/"
+        static let adPolicyLink = "https://automattic.com/advertising-policy/"
+        static let learnMoreLink = "https://wordpress.com/support/promote-a-post/"
+    }
+
+    enum Localization {
+        static let title = NSLocalizedString(
+            "blazeConfirmPaymentView.title",
+            value: "Payment",
+            comment: "Title of the Payment view in the Blaze campaign creation flow"
+        )
+        static let submitButton = NSLocalizedString(
+            "blazeConfirmPaymentView.submitButton",
+            value: "Submit Campaign",
+            comment: "Action button on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let paymentTotals = NSLocalizedString(
+            "blazeConfirmPaymentView.paymentTotals",
+            value: "Payment totals",
+            comment: "Section title on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let blazeCampaign = NSLocalizedString(
+            "blazeConfirmPaymentView.blazeCampaign",
+            value: "Blaze campaign",
+            comment: "Item to be charged on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let total = NSLocalizedString(
+            "blazeConfirmPaymentView.total",
+            value: "Total",
+            comment: "Title of the total amount to be charged on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let totalAmount = NSLocalizedString(
+            "blazeConfirmPaymentView.totalAmount",
+            value: "%1$@ USD",
+            comment: "The formatted total amount for a Blaze campaign, fixed in USD. " +
+            "Reads as $11 USD."
+        )
+        static let addPaymentMethod = NSLocalizedString(
+            "blazeConfirmPaymentView.addPaymentMethod",
+            value: "Add a payment method",
+            comment: "Button for adding a payment method on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let loading = NSLocalizedString(
+            "blazeConfirmPaymentView.loading",
+            value: "Loading payment methods...",
+            comment: "Text for the loading state on the Payment screen in the Blaze campaign creation flow"
+        )
+        static let agreement = NSLocalizedString(
+            "blazeConfirmPaymentView.agreement",
+            value: "By clicking \"Submit campaign\" you agree to the %1$@ and " +
```

Comment:
Nit: Use "Campaign" to match the CTA button title. 
```suggestion
            value: "By clicking \"Submit Campaign\" you agree to the %1$@ and " +
```